tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid_test.py

```diff
@@ -0,0 +1,122 @@
+# Python imports
+import unittest
+
+# Third party imports
+from jinja2 import Template
+from googleapiclient.errors import HttpError
+
+# Project imports
+import bq_utils
+import common
+from cdr_cleaner import clean_cdr_engine
+from cdr_cleaner.cleaning_rules import repopulate_person_post_deid
+from tests import test_util
+
+# Participant 1: has gender and sex at birth observations
+# Participant 2: no gender or sex at birth observations
+INSERT_FAKE_PARTICIPANTS_TMPLS = [
+    # TODO(calbach): Ideally these tests should not manipulate concept table, not currently hermetic.
+    Template("""
+INSERT INTO `{{project_id}}.{{dataset_id}}.concept` (concept_id, concept_code)
+VALUES
+  ({{gender_concept_id}}, "gender"),
+  ({{gender_nonbinary_concept_id}}, "nonbinary"),
+  ({{gender_nonbinary_source_concept_id}}, "nonbinary_src"),
+  ({{sex_at_birth_concept_id}}, "sex"),
+  ({{sex_female_concept_id}}, "female"),
+  ({{sex_female_source_concept_id}}, "female_src")
+"""),
+    Template("""
+INSERT INTO `{{project_id}}.{{dataset_id}}.person` (person_id)
+VALUES (1), (2)
+"""),
+    Template("""
+INSERT INTO `{{project_id}}.{{dataset_id}}.observation` (person_id, observation_id, observation_source_concept_id, value_as_concept_id, value_source_concept_id)
+VALUES
+  (1, 100, {{gender_concept_id}}, {{gender_nonbinary_concept_id}}, {{gender_nonbinary_source_concept_id}}),
+  (1, 101, {{sex_at_birth_concept_id}}, {{sex_female_concept_id}}, {{sex_female_source_concept_id}})
+""")
+]
+
+
+class RepopulatePersonPostDeidTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def assertPersonFields(self, person, want):
+        for (k, v) in want.items():
+            self.assertIn(k, person)
+            self.assertEqual(person[k], want[k])
+
+    def test_execute_queries(self):
+        project_id = bq_utils.app_identity.get_application_id()
+        dataset_id = bq_utils.get_combined_dataset_id()
+        self.assertIsNotNone(project_id)
+        self.assertIsNotNone(dataset_id)
+        test_util.delete_all_tables(dataset_id)
```

Comment:
On the same lines, setting these variables could be included as `self.project_id` etc. in setUp() and delete tables could go in tearDown()

Reply:
Done